### DISCUSSION TITLE
Do the img_size of SSL pretrained model and down stream task model need to be the same?

### DISCUSSION THREAD

**Discussion by justinzhang528** [Project-MONAI/tutorials#1410] [Q&A]
Dear MONAI community,
Because of my gpu memory limited, the img_size of my SSL pretrained network can't larger than 96x96x96，if i trained SSL model with this size and then is it possible to use different img_size in my down stream task?

Thanks in advanced!

**Comment by KumoLiu** [comment]
Hi @justinzhang528, sure you can.
Thanks!

**Reply by justinzhang528** [reply]
got it, thanks!
